README.md

```diff
@@ -1 +1,183 @@
-# Module_1_Hello_World
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="Kamran.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello World</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello World <a name="about-project"></a>
+
+> My first Hello World project for understanding of linters and github flow. I created it using HTML, CSS
+
+
+## 🛠 Built With <a name="built-with"></a>
+1- HTML.
+2- CSS.
+3- linters.
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added index.html file
+- Added style.css file
+- Added text.md file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming soon.
+
```

Comment:
- [ ] As your application is not yet deployed and not having the live demo yet, so kindly delete this section from here and from the table of contents as well, so that no section is left empty and it will give a better reading experience. You can add this section later when you will be having the live demo link to your project.😊